plaso/storage/file_interface.py

```diff
@@ -862,17 +867,24 @@ def PrepareMergeTaskStorage(self, task):
     if self._storage_type != definitions.STORAGE_TYPE_SESSION:
       raise IOError('Unsupported storage type.')
 
-    merge_storage_file_path = self._GetMergeTaskStorageFilePath(task)
-    processed_storage_file_path = self._GetProcessedStorageFilePath(task)
+    if task.storage_format == definitions.STORAGE_FORMAT_REDIS:
+      task.storage_file_size = 1000
+      redis_store.RedisStore.MarkTaskAsMerging(
+          task.identifier, self._session.identifier)
+      return
 
-    task.storage_file_size = os.path.getsize(processed_storage_file_path)
+    if task.storage_format == definitions.STORAGE_FORMAT_SQLITE:
```

Comment:
maybe check for supported session storage formats, error if not and change this into `else:` and drop the `return` above?

Reply:
```
    if task.storage_format not in definitions.TASK_STORAGE_FORMATS:
      raise IOError('Unsupported storage format.')
```